src/library.js

```diff
@@ -4446,34 +4446,9 @@ LibraryManager.library = {
       );
   },
 
-  // Returns [parentFuncArguments, functionName, paramListName]
+  // Function disabled
   _emscripten_traverse_stack: function(args) {
-    if (!args || !args.callee || !args.callee.name) {
-      return [null, '', ''];
-    }
-
-    var funstr = args.callee.toString();
-    var funcname = args.callee.name;
-    var str = '(';
-    var first = true;
-    for (var i in args) {
-      var a = args[i];
-      if (!first) {
-        str += ", ";
-      }
-      first = false;
-      if (typeof a === 'number' || typeof a === 'string') {
-        str += a;
-      } else {
-        str += '(' + typeof a + ')';
-      }
-    }
-    str += ')';
-    var caller = args.callee.caller;
-    args = caller ? caller.arguments : [];
-    if (first)
-      str = '';
-    return [args, funcname, str];
+    return null;
```

Comment:
Thanks for the PR!

Looking earlier up, it looks like the function returns `[null, '', '']` if it can't do better, so maybe that's what we should be returning here? In any case, though, I think we should remove this function, and make the function calling it not try to call it.

Reply:
I attempted to remove the function any instance of it being called